### DISCUSSION TITLE
Code Access

### DISCUSSION THREAD

**Discussion by lonnieCross4568** [primefaces/community#1111] [PrimeNG Blocks] [answered]
I have purchased a license from Prime Blocks, but I can't access the code button on the page. I don't see a login option anywhere on the page. Can someone please help me with this?

![image](https://github.com/primefaces/community/assets/116767136/144c1cb1-e963-463e-98d8-0e9efcc2790c)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You need to download from the store https://www.primefaces.org/store/. If you don't have primefaces store account, please send an e-mail to contact@primetek.com.tr
